pynumaflow/function/udfunction_pb2_grpc.py

```diff
@@ -0,0 +1,170 @@
+# Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
```

Comment:
Do we need this to be in git @vigith ?

Reply:
yes, we need to check this in, or else we might see unexpected behaviors at runtime because it might diverge from the local setup.

Reply:
Is it possible to have these files in a separate `generated` directory?